amundsenatlastypes/schema/05_table_partition_schema.json

```diff
@@ -1,29 +1,27 @@
 {
   "entityDefs": [
     {
+      "name": "TablePartition",
       "superTypes": [
-        "TablePartition"
-      ],
-      "name": "hive_table_partition",
```

Comment:
why are we removing hive_table_partition ? I think that different systems might also have partitioned tables so we should distinguish those the same way we have hive_table and hbase_table deriving from Table

Reply:
again - it is still there but in different file.